web/ui-go/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/enola-dev/enola/ui-go
```

Comment:
I'm myself not yet familiar with https://github.com/bazelbuild/rules_go, but I suspect that with a Bazel file here, a `go.mod` may not be required at all anymore, because the dependencies will be declared in the (new, TBD) `BUILD` file? @digitalentity might know more about this...